src/components/Siderbar.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import {
```

Comment:
Can you replace this file with the official sidebar which have links and translated texts

Reply:
ok, let do that.

Reply:
solved